fedmsg/config.py

```diff
@@ -254,8 +254,10 @@ class FedmsgConfig(dict):
             'validator': None,
         },
         'relay_inbound': {
-            'default': u'tcp://127.0.0.1:2001',
-            'validator': _validate_none_or_type(six.text_type),
```

Comment:
It doesn't make much sense to have a relay binding to multiple sockets. Did you run into this by trying to specify a list of values?

Reply:
I did not run this using multiple entries.

I think the issue is mainly documentation related. 
* The existing config show that parameter as a list (https://github.com/fedora-infra/fedmsg/blob/develop/fedmsg.d/relay.py#L36)
* The `fedmsg-config` command also shows this option as a list. 
* The online docs reference it as a list too (http://fedmsg.readthedocs.io/en/stable/configuration/#relay-inbound)

In my case, I just updated the example config and things worked fine. I never needed to add multiple values to this.

It is very likely I am wrong and reading the wrong documentation parts.

Reply:
Ah, well the history here is I wrote documentation. There wasn't any documentation before and I took wild guesses. Recently I tried to add defaults to every setting and picked what I thought were reasonable defaults.

Although it looks like lists and strings are both handled correctly, it's not a setting that makes a bunch of sense as a list so I'm in favor of changing the documentation and config in ``fedmsg.d`` (really, that directory should go away, but I have no doubt that'd break something weird relying on it 😢 )

Reply:
I will definitely change/revert that list change in this PR.
I could also include documentation changes so that it references it as a string, not list. This would keep everything consistent, at least.

At quick check, tests fail removing the `fedmsg.d` directory. I bet packaging would have issues too. Although, both of those could be solved.

Reply:
I've removed this commit completely.